examples/systemcalls/systemcalls.c

```diff
@@ -93,6 +223,64 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *
 */
 
+   if(check_abs_path(command[0])==false)
+    {
+        syslog(LOG_ERR, "Path must be an absolute path \n");
+        va_end(args);
+        return false;
+    }
+    pid_t pid = fork();
```

Comment:
@shruthi-thallapally think about how you could avoid duplicating logic between this function and the one directly above it.